doc/lua_api.txt

```diff
@@ -4239,6 +4244,32 @@ Definition tables
         ^ Shall drop item and return the leftover itemstack
         ^ The dropper may be any ObjectRef or nil.
         ^ default: minetest.item_drop ]]
+        on_pickup = func(item_luaentity, picker, time_from_last_punch, tool_capabilities, dir),
+        --[[
+        ^ Called when a dropped item is punched and picked up by `picker`.
+        ^ Shall remove dropped item and add itemstack to `picker`'s inventory or
```

Comment:
Do you mean that the function should either return 'true', or itself do the job of removing the object and adding it to the picker's inventory (instead of existing code)?

'Shall' is unclear English, 'should' is better, and a comma is good for scope:
`Should remove dropped item and add itemstack to `picker`'s inventory, or return `true`.`